AppCenter/AppCenterTests/MSACUtilityTests.m

```diff
@@ -156,6 +162,47 @@ - (void)testNowInMilliseconds {
   XCTAssertGreaterThan(actual, 0);
 }
 
+- (void)testDateFormatterConcurrentInitialization {
+
+  // If
+  [MSACUtility resetDateFormatterInstance];
+  XCTestExpectation *expectation = [self expectationWithDescription:@"queueExpectation"];
+  dispatch_queue_t concurrentQueue = dispatch_queue_create("com.dateformatter.queue", DISPATCH_QUEUE_CONCURRENT);
+  id nsDateFormatter = OCMClassMock([NSDateFormatter class]);
+  OCMStub([nsDateFormatter stringFromDate:OCMOCK_ANY]).andReturn(@"stub");
+  OCMStub([nsDateFormatter alloc]).andReturn(nsDateFormatter);
+
+  // When
+  int dispatchTimes = 10;
+  __block NSObject *lock = [NSObject new];
+  __block int counter = 0;
+  for (int i = 0; i < dispatchTimes; i++) {
+    dispatch_async(concurrentQueue, ^{
+      @try {
+        [MSACUtility dateToISO8601:[NSDate dateWithTimeIntervalSince1970:i]];
+      } @catch (NSException *exception) {
+        XCTFail(@"Expectation Failed with error: %@", exception);
+      } @
```

Comment:
@?

Reply:
`./clang-format-changed-files.sh` did it for some reason. I haven't noticed.
The interesting thing is that it works this way :)